test/com/facebook/buck/android/AndroidPrebuiltAarIntegrationTest.java

```diff
@@ -16,6 +16,10 @@
 
 package com.facebook.buck.android;
 
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.not;
```

Comment:
Well, I really just need to get some decent CI up here so machines can tell us when it's broken :)